src/interface_py/h2o4gpu/solvers/truncated_svd.py

```diff
@@ -248,7 +267,7 @@ class TruncatedSVD(object):
 
     def __init__(self,
                  n_components=2,
-                 algorithm="cusolver",
+                 algorithm="power",
```

Comment:
I would leave as `cusolver` to keep consistent with previous version.

Reply:
If power is going to be generally better, should use that as default.  Can't expect users to change defaults.